### DISCUSSION TITLE
请问icons配置可以自定义读取的svg路径吗

### DISCUSSION THREAD

**Discussion by lzq980604** [umijs/umi#11180] [Q&A]
默认读取的是src/icons 这个路径下的svg，如果我想读取src/assets下的svg该怎么修改路径

**Comment by fz6m** [comment]
目前的 `src/icons` 约定只能放到这个文件夹下面，如果是其他文件的 svg 就直接导入使用吧，或者移动到 icons 文件夹里面。

**Reply by lzq980604** [reply]
好的，谢谢回复
